backend/workflow_manager/endpoint_v2/destination.py

```diff
@@ -182,9 +183,9 @@ def handle_output(
         if connection_type == WorkflowEndpoint.ConnectionType.FILESYSTEM:
             self.copy_output_to_output_directory()
         elif connection_type == WorkflowEndpoint.ConnectionType.DATABASE:
-            if (
-                file_hash.file_destination
-                == WorkflowEndpoint.ConnectionType.MANUALREVIEW
+            result = self.get_result(file_history)
+            if WorkflowUtil.validate_db_rule(
```

Comment:
[MINOR] Will the placeholder be overridden at any point?

Reply:
@harini-venkataraman refer cloud PR. yes, cloud utils will be used once available.